src/browser/BrowserSettingsWidget.cpp

```diff
@@ -257,14 +327,26 @@ void BrowserSettingsWidget::showProxyLocationFileDialog()
 #else
     QString fileTypeFilter(QString("%1 (*)").arg(tr("Executable Files")));
 #endif
-    auto proxyLocation = QFileDialog::getOpenFileName(this,
-                                                      tr("Select custom proxy location"),
-                                                      QFileInfo(QCoreApplication::applicationDirPath()).filePath(),
-                                                      fileTypeFilter);
 
-    proxyLocation = browserSettings()->replaceHomePath(proxyLocation);
-    m_ui->customProxyLocation->setText(proxyLocation);
-    validateCustomProxyLocation();
+    QString sInitialFilePath = m_ui->customProxyLocation->text();
```

Comment:
A lot of variables starts with `s` even if it's clear we are handling a string. Maybe use simpler variable names in every class?

Reply:
Removed the excessive `s` in variable name.
Code is equivalent to the code which was in the reviewed pull request.